dashboards-observability/public/components/application_analytics/helpers/utils.tsx

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Name validation 0>Name<=50
+export const isNameValid = (name: string) => {
+  return name.length >= 50 || name.length === 0 ? false : true;
```

Comment:
it might be better to return an array of error messages to tell user why it's not valid, in case we add  conditions other than string length like "allowed characters" in the future

Reply:
Got it!